lib/react_on_rails/server_rendering_pool.rb

```diff
@@ -1,62 +1,20 @@
-require 'connection_pool'
+require "connection_pool"
 
-# Based on the react-rails gem
+# Based on the react-rails gem.
+# None of these methods should be called directly.
+# See app/helpers/react_on_rails_helper.rb
 module ReactOnRails
   class ServerRenderingPool
     def self.reset_pool
       options = { size: ReactOnRails.configuration.server_renderer_pool_size,
                   timeout: ReactOnRails.configuration.server_renderer_pool_size }
-      @@js_context_pool = ConnectionPool.new(options) { create_js_context }
+      @js_context_pool = ConnectionPool.new(options) { create_js_context }
```

Comment:
Changed to using the class instance variable. I think this would only make a difference if there was subclassing, but this is the linters preferred way. @samnang, @rstudner, @alexfedoseev Agree?